gulpfile.esm.js

```diff
@@ -97,7 +94,7 @@ function cordovaCompile() {
   const compileArgs = platform === "ios" ? "--device" : "";
   let releaseArgs = "";
   if (buildMode === "release" && platform === "android") {
-    releaseArgs = `--release -- --keystore=${KEYSTORE} --storePassword=${STOREPASS} --alias=${KEYALIAS} --password=${KEYPASS}`;
+    releaseArgs = `--release -- --keystore=keystore.p12 --alias=privatekey --storePassword=${androidKeyStorePassword} --password=${androidKeyStorePassword}`;
```

Comment:
Let's put the keystore in a private repo, not in the main repo.

Reply:
Okay - i'm confused then, how do you want to do this? I thought we were against `outline-credentials` as a repo